.github/workflows/master.yml

```diff
@@ -169,7 +169,7 @@ jobs:
           git commit -m "Update documentation (libevent/libevent@$short_commit_id)"
           git push -f git@github.com:$owner_name/doc master
 
-      - uses: actions/upload-artifact@3446296876d12d4e3a0f3145a3c87e67bf0a16b5 # v1.0.0
```

Comment:
We have a dependabot to update this dependencies, see https://github.com/libevent/libevent/pull/1590

Reply:
Let's give dependabot a chance. I'll close this PR and we see what happens in the upcoming 2 weeks.

Reply:
Since dependabot has been unsatisfying (it barely updates anything in #1639), I'm reopening my PR and updating it with 21f7df590c27b532088949e38054a62a4e7b6bf1